packages/spotify-client-deb/spotify-client-deb.pacscript

```diff
@@ -1,9 +1,11 @@
-name='spotify-client-deb'
-gives='spotify-client'
-pkgname='spotify-client'
-breaks="${pkgname} ${pkgname}-git ${pkgname}-app ${pkgname}-bin spotify"
-version='1.1.72.439.gc253025e'
-url="http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_${version}_amd64.deb"
-description='Spotify streaming music client'
-hash='c211a16b6ed6627c4b7d8ca6997376f8345dca638938696933c5495405ce9e16'
-maintainer='TwilightBlood <hwengerstickel@pm.me>'
+name="spotify-client-deb"
+gives="spotify-client"
+pkgname="spotify-client"
+repology=("project: spotify")
+breaks="${pkgname}-git ${pkgname}-app ${pkgname}-bin spotify"
+replace="${pkgname}"
+version="1.1.80.699.gc3dac750"
+url="http://repository.spotify.com/pool/non-free/s/${gives}/${gives}_${version}_amd64.deb"
+description="Spotify streaming music client"
+hash="19f80255b89d768969ff9d6a05d90091a30c05f65eabdd968592d47a7754b80e"
+maintainer="TwilightBlood <hwengerstickel@pm.me>"
```

Comment:
@Henryws, so you're maintaining this in the future right?

Reply:
Ah, I think I made that at someones request, iirc. No, I don't wish to maintain it as I don't use spotify.

Reply:
@WRM-42 do you want to take over maintaining this?